src/profile/v100/networks/network/authorized_dapp/authorized_persona_simple.rs

```diff
@@ -79,41 +79,65 @@ impl Identifiable for AuthorizedPersonaSimple {
     }
 }
 
+impl IsNetworkAware for AuthorizedPersonaSimple {
+    fn network_id(&self) -> NetworkID {
+        self.identity_address.network_id()
+    }
+}
+
+impl PersonaData {
+    pub(crate) fn shared_everything(&self) -> SharedPersonaData {
+        SharedPersonaData::new(
+            self.name.clone().map(|x| x.id),
+            SharedToDappWithPersonaIDsOfPersonaDataEntries::exactly(
+                self.email_addresses.clone().ids(),
+            ),
+            SharedToDappWithPersonaIDsOfPersonaDataEntries::exactly(
+                self.phone_numbers.clone().ids(),
+            ),
+        )
+    }
+}
+
 impl AuthorizedPersonaSimple {
     pub fn sample_mainnet() -> Self {
```

Comment:
make sure that the data in `SharedPersonaData` matches the data in `persona.persona_data` for all sample values.